etc/openqa/openqa.ini

```diff
@@ -107,6 +107,8 @@
 #do_cleanup = no
 # whether committed changes should be pushed
 #do_push = no
+# whether to clone CASEDIR or NEEDLES_DIR on the web UI if that var points to a Git repo
+#auto_clone = no
```

Comment:
```suggestion
#git_auto_clone = no
```

Reply:
We already had this: In my opinion `auto_clone` makes more sense because this is already under the `scm git` section.